### ISSUE TITLE
How to execute a POST request? 

### ISSUE THREAD

**Issue by vvpnet** [oatpp/oatpp#652]
Hello. I need to run a simple query. For example, how do I do it through curl:
curl -i -k POST "http://localhost:8080/" -H "Content-Type: application/x-www-form-urlencoded" -d client_id=123 -d client_secret=secret123 -d username=user -d password=paswd -d type=sample

And on the way out it looks like this:
POST / HTTP/1.1
Host: localhost:8080
User-Agent: curl/7.58.0
Accept: */*
Content-Type: application/x-www-form-urlencoded
Content-Length: 163

client_id=123&client_secret=secret123&username=user&password=paswd&type=sample

I tried it like this:

  API_CALL("POST", "/test", getTest,
           HEADER(String, content, Header::CONTENT_TYPE),
           BODY_STRING(String, client_id, "client_id"),
           BODY_STRING(String, client_secret, "client_secret"),
           BODY_STRING(String, username, "username"),
           BODY_STRING(String, password, "password"),
           BODY_STRING(String, type, "type"))

And I always get this result:

POST /test HTTP/1.1
Host: localhost:8080
Accept: */*
Content-Type: application/x-www-form-urlencoded

I use oatpp-curl.

**Comment by lganzzzo** [comment]
Hello @vvpnet ,

You should have only one body in the request.
It should look something like the following:

```cpp
API_CALL("POST", "/test", getTest,
         HEADER(String, content, Header::CONTENT_TYPE),
         BODY_STRING(String, content))
```

Then you should manually construct your content:

```cpp
oatpp::String content = "client_id=123&client_secret=secret123&username=user&password=paswd&type=sample";

auto result = myApiClient.getTest("application/x-www-form-urlencoded", content);
```

**Comment by vvpnet** [comment]
Yes, I've already tried that. The result remains the same for oatpp-curl (1.3.0). But for oatpp::web::client::HttpRequestExecutor is a completely different behavior. The result was as follows:
```
POST /test HTTP/1.1
Content-Length: 163
Connection: keep-alive
Content-Type: application/x-www-form-urlencoded
Host: localhost

client_id=123&client_secret=secret123&username=user&password=paswd&type=sample
```

It turns out oatpp-curl(1.3.0) has problems.

**Comment by lganzzzo** [comment]
Sorry, I think I'm not fully understand...

- What's the difference in requests produces by `oatpp::web::client::HttpRequestExecutor` and oatpp-curls executors?
- Which of the executors works correctly and which of them works incorrectly and in what way?
